tendrl/monitoring_integration/grafana/alert.py

```diff
@@ -1,25 +1,51 @@
+import json
 import maps
 
 from requests import get
 from requests.exceptions import ConnectionError
 from requests.exceptions import RequestException
+
 from tendrl.commons.utils import log_utils as logger
 from tendrl.monitoring_integration.grafana import exceptions
+from tendrl.monitoring_integration.grafana import grafana_org_utils
 from tendrl.monitoring_integration.grafana import utils
 
 
+def connect(alert_id, config):
+    resp = get("http://{0}:{1}/api/alerts/"
+               "{2}".format(config["grafana_host"],
+                            config["grafana_port"],
+                            
+                            alert_id),
+               auth=config["credentials"])
+    return resp
+
+def switch_context(org_name):
+    alert_org_id = grafana_org_utils.get_org_id(
+        org_name
+    )
+    grafana_org_utils.switch_context(json.loads(
```

Comment:
Switch context returns true and false which confirms whther the switch was successful or not. So I think you should check that before proceeding to other actions assuming that it has switched.

Reply:
ok